packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/SVGImage/SVGImage.js

```diff
@@ -35,9 +35,11 @@ function SVGImageWithRef({
 
   const props = {
     'aria-label': `Subject ${subjectID}`,
+    draggable: false,
```

Comment:
```suggestion
    draggable: ‘false’,
```

Reply:
The HTML `draggable` attribute is a string, not a boolean. 

`element.draggable` is a boolean. I’m not sure which one React uses.

Reply:
React uses the boolean DOM property, which renders as `draggable='false'` in the output HTML.